src/internal/components/dataset/NamedSetList.tsx

```diff
@@ -1,22 +1,26 @@
 import React from 'react';
 import {Button, ButtonGroup, Col, Dropdown} from 'react-bootstrap';
-import {INamedSet} from 'tdp_core';
+import {ENamedSetType, FormDialog, INamedSet, IStoredNamedSet, NotificationHandler, RestStorageUtils, StoreUtils} from 'tdp_core';
 import {SESSION_KEY_NEW_ENTRY_POINT} from '../..';
 import {ListItemDropdown} from '../common';
 import {GraphContext} from '../../OrdinoAppComponent';
-import {UserSession} from 'phovea_core';
+import {I18nextManager, UserSession} from 'phovea_core';
+import {DropdownItemProps} from 'react-bootstrap/esm/DropdownItem';
 
 interface INamedSetListProps {
   headerIcon: string;
   headerText: string;
-  namedSets: INamedSet[] | null;
+  value: INamedSet[] | null;
```

Comment:
It looks like the `onOpen` function from https://github.com/Caleydo/ordino/pull/272/files#diff-b52d524c04e27a53700c615368dd36dcb6e0aece270cd891efdb2f9c14384d64R10 got lost. Please re-add and use in the tdp_publicdb as before. Thanks.